apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -809,7 +816,12 @@ aggregate_metrics(
         M15 + N15,
         M16 + N16,
         M17 + N17
-    ).
+    );
+aggregate_metrics(
+    _FailedNode,
```

Comment:
after the filter,  is this still reachable? 

having a clause like this may cause silent drops of metrics if in the future we decided to add a new dimension, say M18. ?

Reply:
OK. I'll convert this check to a filter to remove failed metrics before calling this function